src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -129,7 +129,9 @@
 				</NcActionButton>
 			</template>
 			<template v-else-if="submenu === 'notifications'">
-				<NcActionButton :aria-label="t('spreed', 'Back')"
+				<NcActionButton
+					key="action-back"
```

Comment:
does this refresh good enough even when its reused inside a parent that has a different key? Just asking for my understanding

Reply:
if parent has a different key, it should re-render parent and everything underneath

Reply:
> does this refresh good enough even when its reused inside a parent that has a different key? Just asking for my understanding

Key doesn't trigger re-rendring, but re-order/re-mount.

And Vue doesn't support parent remounting without children remounting.

So changing the parent key completely destroys and re-creates children. Not just re-render.